pytorch_lightning/callbacks/progress/tqdm_progress.py

```diff
@@ -256,14 +273,15 @@ def on_sanity_check_start(self, *_: Any) -> None:
 
     def on_sanity_check_end(self, *_: Any) -> None:
         self.main_progress_bar.close()
+        self.main_progress_bar = None
```

Comment:
Can you add a comment for why we dereference the main_progress_bar?

Reply:
removed the dereference.